tests/consensus_tests/test_replicate_points.py

```
@@ -1,9 +1,11 @@
 import pathlib
+from time import sleep
 
 from .fixtures import upsert_random_points, create_collection
 from .utils import *
```

Ruff (0.14.2)

`from .utils import *` used; unable to detect undefined names

(F403)